cmake/win32.cmake

```diff
@@ -24,7 +24,7 @@ if(NOT MSVC_VERSION)
   add_definitions(-DWINVER=0x0500 -D_WIN32_WINNT=0x0500)
   # Wait a minute, if we're not Microsoft C++, nor a Clang paired with Microsoft C++,
   # then the only possible option has to be GNU or a GNU-linked Clang!
-  set(FS_LIB stdc++fs)
+  # set(FS_LIB stdc++fs)
```

Comment:
but why tho

Reply:
Latest mingw ships with a non-broken stdlib

Reply:
ok apparently only mainstream targets get this special treatment

t. builds on a bunch of niche, arcane, yet up to date platforms that still ship a separate c++17 filesystem ts lib

Reply:
wrap it around a library check or version check at least